db/migrate/20150728014635_create_pieces.rb

```diff
@@ -0,0 +1,13 @@
+class CreatePieces < ActiveRecord::Migration
+  def change
+    create_table :pieces do |t|
+    	t.string  :type   
+    	t.integer :active
+    	t.text    :square_location
+    	t.integer :game_id
```

Comment:
For this column, you might want to create it with an index.

Rails has a shorthand for doing this quickly.

`t.references :game, index: true`

This will create the column `game_id` and create an index for it.

More info on indexes: https://tomafro.net/2009/08/using-indexes-in-rails-index-your-associations (this is a little old, so the syntax is different, but the overall concept is described very well)